cmd/commands.go

```diff
@@ -170,6 +169,9 @@ var gbHelpCmd = &cobra.Command {
 	Args:					cobra.RangeArgs(0, 1),
 }
 
+//gbHelpFunc takes a pointer to cobra.command and
+//command arguments and output help message
+
```

Comment:
@kazishafin — You cannot have a blank line after the comment for a function and the function itself.